server/utils/config.py

```diff
@@ -0,0 +1,5 @@
+
+import configtools
+
+opts, _ = configtools.parse_args()
```

Comment:
What do we gain making the `opts` and `conf` be module level variables?

Reply:
Just separation of concerns for code readability and potential use on additional libraries.

Reply:
Could we consider merging this convenience function into the `lib/configtools` module entirely, adding something like the following there?

```
class _SimpleOpts(object):
    def __init__(self, conf_file=None):
        self.filename = conf_file
def get_conf(conf_file=None):
    conf, _ = init(_SimpleOpts(conf_file), '_PBENCH_SERVER_CONFIG')
    return conf
```

This way we use the sample server config environment variable, and we avoid parsing options intended for the `getconf.py` interfaces, and unit tests can then pass an explicit configuration file for testing.

Reply:
Another option for the server side is to just use the `server/lib/pbench/__init__.py:PbenchConfig` object itself.

Is there a reason why that wouldn't work?